app/src/pages/review/[resourceType]/[id].tsx

```diff
@@ -100,6 +102,7 @@ export default function CommentPage() {
   function parseUpdate(comment: string, type: string, userName: string, dateSelected: boolean) {
     const additions: DraftArtifactUpdates = {};
     const deletions: DraftArtifactUpdates = {};
+    const currentDate = new Date().toISOString();
```

Comment:
This is what we had before so not a big deal at all since I am just noticing, but is this the time that we want?